src/RecoveryServices/RecoveryServices.SiteRecovery/DiskReplicationConfiguration/AzureRmAsrAzureToAzureDiskReplicationConfig.cs

```diff
@@ -84,7 +84,7 @@ public class AzureRmAsrAzureToAzureDiskReplicationConfig : SiteRecoveryCmdletBas
         [ValidateSet(
             Constants.Premium_LRS,
             Constants.Standard_LRS,
-            Constants.Standard_SSD)]
+            Constants.StandardSSD_LRS)]
```

Comment:
change the validate set of mandatory parameters is not allowed in non-breaking change release, you can add the "Constants.StandardSSD_LRS" without removing "Constants.Standard_SSD" 

then 

convert Constants.Standard_SSD to Constants.StandardSSD_LRS underlying

Reply:
updated as suggested.